problems/variables.js

```diff
@@ -2,48 +2,112 @@
 
 // Age Calculator:
 //  * Store your birth year in a constant variable.
+const birthYear = 1992
+
+
 //  * Store a future year in a variable.
+
+let futureYear = 2030
+
 //  * Calculate your 2 possible ages for that year based on the stored values.
+ 
+console.log(futureYear-birthYear)
+"I will be either '37' or '38' in '2030'"
+
+
 //  * For example, if you were born in 1988, then in 2026 you'll be either 37 or 38, depending on what month it is in 2026.
 //  * Log them to the screen like so: "I will be either `ageMin` or `ageMax` in `futureYear`", substituting the values.
 
 // ## Problem Two
 
 // Snack Supply Calculator:
 //  * Store your current age in a variable.
+let myAge = 28
+
 //  * Store a maximum age in a constant variable.
+const maxAge = 100
+
 //  * Store an estimated snack amount per day (as a number).
+let dailySnack = 3
+
 //  * Calculate how many snacks you would eat total, from your current age until the maximum age.
+let year1 = 365
+console.log(maxAge - myAge)
+let totalAge = 72
+console.log(totalAge * year1)
+let totalDays = 26280
+console.log(dailySnack * totalDays)
+let snackNumber = 78840
+
 //  * Log the result to the screen like this: "You will need `snackNumber` to last you until the age of `maxAge`".
+"you will need '7840' to last you until the age of '100'"
 
 // ## Problem Three
 
 // Calculate properties of a circle, using the definitions: http://math2.org/math/geometry/circles.htm 
 //  * Store a radius into a variable.
+let radius = 2
+let diamater = 4
+let pi = 3.14
+console.log(pi * diamater)
+
 //  * Calculate the circumference based on the radius, and log "The circumference is `circumferenceResult`".
+"the circumference is '12.56'"
+
 //  * Calculate the area based on the radius, and log "The area is `areaOfCircle`".
+let radiusSquared = 4
+console.log(pi * radiusSquared)
+
+"the area is '12.56'"
+
 //  * Hint: https://www.w3schools.com/jsref/jsref_pi.asp
 
 // ## Problem Four
 
 // Temperature Converter:
 //  * Store a celsius temperature into a variable.
+let celciusTemp = 5
+
 //  * Convert it to fahrenheit and output "`tempInCelsius`°C is `tempInFahrenheit`°F".
+"'5°C' is 41°F'"
+
 //  * Now store a fahrenheit temperature into a variable.
+
+let fahreinheitTemp = 80
+
 //  * Convert it to celsius and output "`tempInFahrenheit`°F is `tempInCelsius`°C."
+"'80°F' is '27°C'"
```

Comment:
One point of clarification here: you want to console log this value, so you'd put something like 

```
console.log(celciusTemp + " is " + fahrenheitTemp)
```

rather than just writing the string on its own